### ISSUE TITLE
High frequency songs play slow

### ISSUE THREAD

**Issue by Emerald-Fox** [Fledge68/WiiFlow_Lite#221]
High frequency OGG files play slowed down and low pitched.
I just updated from v5.4.6 to v5.4.8 and the songs played fine before I updated, 44100 Hz still plays normal though.

**Comment by Hakaisha** [comment]
Emerald-Fox, I have been trying to reproduce your issue, but have so far been unsuccessful. 

Every 48000 Hz ogg I've tested with WFL 5.4.8 plays as smoothly as the 44100 Hz ogg files. I've tried exporting from both Audacity and NCH Switch, acheiving the same results in both 5.4.6 and 5.4.8.

In my testing of WFL's music player, I have experienced many slow and low pitched issues with MP3 files. The issue was usually much more prevalent in songs with a higher kbps rate, regardless of the Hz rate.

Do you notice any marked difference in the kbps between the problematic oggs and the properly functioning oggs? Also, how high is the frequency of the problematic oggs?

**Comment by Emerald-Fox** [comment]
The oggs are over 48000 Hz, 96000 Hz to be exact, also the kbps is dynamic for both the functioning and non functioning oggs.

**Comment by Hakaisha** [comment]
I tried converting some songs to 96 KHz OGGs, and it is as you describe.

- WFL 5.4.8 plays the 96 KHz OGGs at a slow speed and are low pitched. (It happens regardless if the 48 KHz setting is on or off)

- WFL 5.4.6 plays them at accurate speed (albeit with slight crackles, compared to their 48 and 44.1 versions)

Fledge added the 'Upsample music to 48KHz' option in WFL 5.4.7, which has markedly improved the sound quality for WFL music player overall.

However, it seems this change may have inadvertently affected music with a higher frequency than 48 KHz. 

Out of curiosity, I also tested out the 96 KHz OGGs with WiiXplorer, and still experience the same slow/low pitch issues, with the 48 KHz option turned on and off.

@Fledge68 , any thoughts on this?
